src/types/profile.js

```diff
@@ -130,7 +130,7 @@ export type RawMarkerTable = {
  * time. The relationship between frames is defined by the StackTable.
  */
 export type FrameTable = {
-  address: IndexIntoStringTable[],
+  address: Array<IndexIntoStringTable | -1>,
```

Comment:
siiiigh

Reply:
I don't remember if I saw this for real, _but_ we account for it in our existing code.